server/client.go

```diff
@@ -2043,15 +2058,26 @@ func (c *client) processInboundClientMsg(msg []byte) {
 		c.checkForImportServices(c.acc, msg)
 	}
 
+	var queuesa [16][]byte
+	queues := queuesa[:0]
+
 	// Check for no interest, short circuit if so.
 	// This is the fanout scale.
 	if len(r.psubs)+len(r.qsubs) > 0 {
-		c.processMsgResults(c.acc, r, msg, c.pa.subject, c.pa.reply)
+		var qnames *[][]byte
```

Comment:
Does not need to be *[], just [][]byte

Reply:
Ah you are passing it around and want it updated in place. Got it.

Reply:
Right.